src/pages/dapi/flutter/index.md

```diff
@@ -117,7 +117,7 @@ class MyApp extends StatelessWidget {
 
 效果是这样的：
 
-![hello-world](img/flutter/Screenshot_0.png)
+![hello-world](img/Screenshot_0.png)
```

Comment:
I cannot put files alongside markdown files. I have to put all images under `/public` since `Astro.resolve()` does not work in markdown files.

This is the first reason I prefer not to switch from jekyll to astro.